platinum.us/res.sha1

```diff
@@ -0,0 +1,341 @@
+42e2b71655a74d6e5bffb9ec75a25df930b20c59 res/battle/moves/pl_waza_tbl.narc
+683a7691d685c153e738b136b8266deefb0b4552 res/items/nuts_data.narc
+70d103831556bb58ae8e602adbcce99333a22889 res/items/pl_item_data.narc
+3614557aa04eb1ad7e5f260578de925f73da584d res/text/pl_msg.narc
+
+1f8689cbc763d9efedac9e6f12e940dbd361f7a9 res/prebuilt/poketool/personal/pl_personal.narc
```

Comment:
Do we have to include the prebuilt narcs? Does it make sense for us to include the prebuilt narcs' shasums?

Reply:
I thought about this again and realized that it would only spit out one `OK` at the end, so I'm changing my mind on this bit: I'm fine with it. I was worried about getting a ton of `OK`s and polluting the output.